include/ringmesh/mesh/mesh_builder.h

```diff
@@ -51,8 +51,13 @@ namespace RINGMesh
     FORWARD_DECLARATION_DIMENSION_CLASS( PointSetMesh );
     FORWARD_DECLARATION_DIMENSION_CLASS( LineMesh );
     FORWARD_DECLARATION_DIMENSION_CLASS( SurfaceMeshBase );
+    FORWARD_DECLARATION_DIMENSION_CLASS( SurfaceMesh );
+    FORWARD_DECLARATION_DIMENSION_CLASS( VolumeMesh );
 
+    struct CellLocalFacet;
     struct EdgeLocalVertex;
+    struct ElementLocalVertex;
+    struct PolygonLocalEdge;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant forward declaration; the declaration was already made. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3231)